ci/nodepool/scripts/prepare_node.sh

```diff
@@ -58,6 +59,9 @@ else
     sudo "${PKG_MGR}" install -y puppet-agent
 fi
 
+echo "Installing git"
```

Comment:
Move down here in order to install after EPEL repo being setup on RHEL5, otherwise not git package will be available.